simde/arm/neon/minv.h

```diff
@@ -34,6 +35,37 @@ HEDLEY_DIAGNOSTIC_PUSH
 SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
 SIMDE_BEGIN_DECLS_
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde_float16_t
+simde_vminv_f16(simde_float16x4_t a) {
+  simde_float32_t r;
+  simde_float16x4_private a_ = simde_float16x4_to_private(a);
+
+  r = simde_float16_to_float32(SIMDE_INFINITYHF);
+  #if defined(SIMDE_FAST_NANS)
+    SIMDE_VECTORIZE_REDUCTION(min:r)
+  #else
+    SIMDE_VECTORIZE
+  #endif
+  for (size_t i = 0 ; i < (sizeof(a_.values) / sizeof(a_.values[0])) ; i++) {
+    simde_float32_t a32 = simde_float16_to_float32(a_.values[i]);
+    #if defined(SIMDE_FAST_NANS)
+      r = a32 < r ? a32 : r;
+    #else
+      r = a32 < r ? a32 : (a32 >= r ? r : ((a32 == a32) ? r : a32));
+    #endif
+  }
+
+  return simde_float16_from_float32(r);
+}
+#if defined(SIMDE_ARM_NEON_A64V8_NATIVE) && defined(SIMDE_ARM_NEON_FP16)
+#define simde_vminv_f16(v) vminv_f16(v)
```

Comment:
The function signature has no constants, please move the native fallback to the main function above (and check elsewhere for the same issue).